packages/playground-ui/src/ds/components/Shimmer/shimmer.css

```
@@ -0,0 +1,33 @@
+/* Sweeping highlight for text that is still being produced.
+ * The gradient period is a fixed pixel width and the animation travels exactly
+ * one period, so the loop never jumps and a short label sweeps at the same
+ * speed as a long one. */
+.shimmer-text {
+  --shimmer-period: 180px;
+
+  background-image: linear-gradient(90deg, var(--neutral3) 0%, var(--neutral6) 50%, var(--neutral3) 100%);
+  background-size: var(--shimmer-period) 100%;
+  background-repeat: repeat;
+  -webkit-background-clip: text;
```

Stylelint (17.14.0)

[error] Vendor-prefixed property "-webkit-background-clip" (property-no-vendor-prefix)

(property-no-vendor-prefix)